.github/workflows/build.yml

```diff
@@ -25,6 +25,7 @@ jobs:
          echo "matrix_rpms=$(jq -c . < ./rpms/config.json)" >> $GITHUB_OUTPUT
          echo "matrix_debs=$(jq -c . < ./debs/config.json)" >> $GITHUB_OUTPUT
          echo "matrix_test=$(jq -c 'del(.include[] | select(.target_arch == "aarch64"))' < rpms/config.json)" >> $GITHUB_OUTPUT
```

Comment:
Do we still need this matrix? Also `test_rpms` action is still using `matrix_test`. Ahh but I guess aarch64 cannot be tested on runners.

Reply:
yes was just a workaround because testing on current runners is really slow, but we should get arm hosted runners before EOY